theme/ckeditor5-image/imageuploadprogress.css

```diff
@@ -8,7 +8,7 @@
 	--ck-image-upload-progress-line-width: 30px;
 }
 
-.ck.image {
+.ck .image {
```

Comment:
I think we should get back to `.ck figure.image`. We replaced `figure` with `.ck` recently, but when we have `.ck .image` we may apply to just too much content.

Reply:
Or let's use `.ck-content .image` as in all other `.image` usage. Let's be consistent with what we have for now and we can consider making this selector safer in the future.